tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
I agree, new tests should be `.c` or `.cpp`, and it would be nice to fix old tests too.

I think the history here is someone wrote a script to pull out sources from the python file, and it wasn't easy to know the right suffix, so we did `.in`.